README.md

```diff
@@ -0,0 +1,168 @@
+    * Use your own container image repository, including `imagePullSecrets` and private repositories  support
+    * Customize your PostgreSQL configuration 
+    * Bring your own trusted certificate authority (CA) for use with the Operator API server
+    * Override your PostgreSQL configuration for each cluster
+    * Use your own custom images, re-define the image for each container separately
+
+## Deployment Requirements
+
+The Operator is validated for deployment on Kubernetes, GKE and EKS clusters. The Operator is cloud native and storage agnostic, working with a wide variety of storage classes, hostPath, and NFS.
+
+The Operator includes various components that get deployed to your Kubernetes cluster as shown in the following diagram and detailed in the Design section of the documentation for the version you are running.
+
+### Supported Platforms
+
+The Operator is tested against the following platforms:
+
+* Google Kubernetes Engine (GKE)
+* Amazon EKS 
+
+Other Kubernetes platforms may also work but have not been specifically tested.
+
+### Storage
+
+The Operator is tested with a variety of different types of Kubernetes storage and Storage Classes, as well as hostPath and NFS.
+The variety of different Storage Classes available for Kubernetes is too wide to verify the Operator functionality in each one. With that said, the Operator is designed to be storage class agnostic and has been demonstrated to work with additional Storage Classes.
+
+## Installation
+
+The following steps will quickly make the Operator up and running with cloud native PostgreSQL:
+
+1. Clone the percona-postgresql-operator repository:
+
+    ```
+    $ git clone  -b release-0.1.0 https://github.com/percona/percona-postgresql-operator
+    $ cd percona-postgresql-operator
+    ```
+
+2. Create the ``pgo`` Namespace for the Operator:
+
+    ```
+    $ kubectl create namespace pgo
+    ```
+
+3. Deploy the operator with the following command:
+
+    ```
+    $ kubectl apply -f ./deploy/operator.yaml
+    ```
+
+4. Percona Server for PostgreSQL cluster can be created with the following command:
+
+    ```
+    $ kubectl apply -f ./deploy/cr.yaml
+    ```
+
+5. You can also deploy PosgreSQL replica at any time as follows: 
+
+    ```
+    $ kubectl apply -f ./deploy/cr-pgreplica.yaml
+    ```
+
+## Design overview
+
+![Percona Kubernetes Operator for PostgreSQL](operator.png)
+
+PostgreSQL containers deployed with the PostgreSQL Operator include the following components:
+
+* [PostgreSQL](https://www.postgresql.org/)
+  * [PostgreSQL Contrib Modules](https://www.postgresql.org/docs/current/contrib.html)
+  * [pgAudit](https://www.pgaudit.org/)
+  * [set_user](https://github.com/pgaudit/set_user)
+  * [wal2json](https://github.com/eulerto/wal2json)
+* [pgBackRest](https://pgbackrest.org/)
+* [pgBouncer](http://pgbouncer.github.io/)
+* [Patroni](https://patroni.readthedocs.io/)
+* LLVM (for JIT compilation)
+
```

Comment:
we need to mention that we also use pg_stat_monitor 
https://github.com/percona/pg_stat_monitor/